### ISSUE TITLE
indentOperator spray doesn't indent after assignment operator

### ISSUE THREAD

**Issue by hseeberger** [scalameta/scalafmt#848]
This template is a guideline, not a strict requirement.
- **Version**: 0.6.6
- **Integration**: sbt
- **Configuration**:
```
style = defaultWithAlign

danglingParentheses        = true
indentOperator             = spray
maxColumn                  = 100
project.excludeFilters     = [".*\\.sbt"]
rewrite.rules              = [RedundantBraces, RedundantParens, SortImports]
spaces.inImportCurlyBraces = true
unindentTopLevelOperators  = true
```

## Problem

Scalafmt formats code like this:
```scala
  val x =
  1 +
  2 +
  3
```

## Expectation

I would like the formatted output to look like this:
```scala
  val x =
    1 +
    2 +
    3
```

**Comment by franbh** [comment]
Any news on this? It still happens when using scalafmt version 1.1.0.

**Comment by ShaneDelmore** [comment]
@franbh It’s still open, with no updates so there is no reason to expect it would have been fixed in 1.1.0.  The ticket has not been closed though which means it is not being ignored, just nobody has found time to work on it yet.  If you have urgent need for it the most expedient way to get the change in is to make a pr yourself (the project is very easy to contribute to in my experience).

**Comment by olafurpg** [comment]
I can reproduce with the following settings

```
indentOperator = spray
unindentTopLevelOperators  = true
```

I suspect the culprit is the convoluted boolean conditions around indentation for infix operators here https://github.com/scalameta/scalafmt/blob/0a5bfb9729932b5d069de889b550d2b7b3ee1c6b/scalafmt-core/shared/src/main/scala/org/scalafmt/internal/FormatOps.scala#L480 

The best way to get this fixed would be to send a PR indeed, my bandwidth to work on scalafmt has been limited past months.

**Comment by olafurpg** [comment]
Opened https://github.com/scalameta/scalafmt/pull/1191 to fix this issue.

**Comment by olafurpg** [comment]
The fix #1191 for this issue has been reverted in #1286 due to it causing a regression https://github.com/scalameta/scalafmt/issues/1223

I'm still keeping this issue closed with "working as expected". There is no clear definition of how `unindentTopLevelOperators=true` should behave in combination with assignments.
```scala
val x = a +
  b
```
The setting has never been documented on the website and unless somebody volunteers to maintain the setting the current behavior will remain unchanged
```scala
val x = a +
b
```
